gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopOutputStreamIntegrationTest.java

```diff
@@ -229,12 +229,14 @@ public void append_shouldFail_whenFileDoesNotExist() throws Exception {
     GoogleJsonResponseException hsyncException =
         assertThrows(GoogleJsonResponseException.class, fsos::hsync);
     assertThat(hsyncException.getStatusCode()).isEqualTo(HttpStatusCodes.STATUS_CODE_NOT_FOUND);
-    assertThat(hsyncException.getMessage()).contains(String.format(" %s ", hadoopPath.getName()));
+    assertThat(hsyncException.getMessage()).contains(String.format("%s", hadoopPath.getName()));
+    assertThat(hsyncException.getMessage()).contains(String.format("%s", "No such object"));
```

Comment:
From where this is coming?

Reply:
I believe you are referring to [PR#1125.](https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1125)